### PR TITLE
[SMALLFIX] [branch-1.3] Remove excessive logging messages w.r.t. metrics system

### PR DESCRIPTION
**Pull request by apc999**
When doing `alluxio runTests`, it outputs to console messages unnecessary. This PR removes these logs.
Especially, the metrics config file is optional
```bash
2016-09-30 11:00:41,442 ERROR type (MetricsConfig.java:loadConfigFile) - Error loading metrics configuration file.
2016-09-30 11:00:41,442 WARN  type (MetricsSystem.java:startSinksFromConfig) - Sinks have already been started.
```

@peisun1115

### CODE REVIEW

File: examples/src/main/java/alluxio/examples/BasicOperations.java [hunk 1]
```diff
@@ -68,7 +68,6 @@ public BasicOperations(AlluxioURI masterLocation, AlluxioURI filePath, ReadType
   public Boolean call() throws Exception {
     Configuration.set(PropertyKey.MASTER_HOSTNAME, mMasterLocation.getHost());
     Configuration.set(PropertyKey.MASTER_RPC_PORT, Integer.toString(mMasterLocation.getPort()));
-    ClientContext.init();
```
Comment (peisun1115) [inline examples/src/main/java/alluxio/examples/BasicOperations.java]: should we remove ClientContext.init() in all places given that it is initialized in the static block of ClientContext class. Moreover, the init function doesn't seem to be threadsafe but ClientContext is annotated as threadsafe.

Comment (apc999) [inline examples/src/main/java/alluxio/examples/BasicOperations.java]: fixed

**Comment by apc999** [comment]
@peisun1115 PTAL

**Comment by peisun1115** [comment]
LGTM

**Comment by apc999** [comment]
@peisun1115 I restored the `ClientContext.init()` in `AbstractFileSystem` as I think it serves a purpose (reinit the master socket), also it might be a sensitive change at this moment for 1.3 before I could fully understand the implication. Are you fine with it?

**Comment by peisun1115** [comment]
LGTM

**Pull request merged**